.gitignore

```diff
@@ -20,3 +20,4 @@ test/manual_scripts/cachetest.go
 lint.log
 azure-storage-fuse
 bfusemon
+test/scripts/dirIterate.go
```

Comment:
what is this ignore file for?

Reply:
This was some local file getting checked-in in all my commits so just added it to the ignore list.